pkg/runtime/retriableError.go

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
I am not sure about the file naming format,  there is file named `initError.go` under the same pkg , but some file located in other pkg use `_` to separate words.

Reply:
should be retriable_error. Feel free to rename initError to init_error too